app/livechat/client/views/app/tabbar/customerChatHistory.js

```diff
@@ -0,0 +1,230 @@
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import _ from 'underscore';
+import './customerChatHistory.html';
+import { APIClient, t } from '../../../../../utils/client';
+import { AccountBox, TabBar, MessageTypes } from '../../../../../ui-utils';
+import {Mongo ,MongoInternals} from 'meteor/mongo';
+import { Tracker } from 'meteor/tracker';
+import { ReactiveDict } from 'meteor/reactive-dict';
+import { Session } from 'meteor/session';
+
+const ITEMS_COUNT = 50;
+let  historyResult 
+let len, msgs
+let allHistoryChat ;
+
+Template.customerChatHistory.helpers({
+	
+	isAllChat:function(){
+        // will return is have to load all chat 
+		return Template.instance().isAllChat.get();
+	},
+	isChatClicked:function(){
+		// will return that if you have clicked in a single chatHistory
+		return Template.instance().isChatClicked.get();
+	},
+	isfound:function() {
```

Comment:
You don't need this.

Reply:
i need it to load the template chatHistoryMessages when user will click on a room